src/ripple/app/ledger/Ledger.cpp

```diff
@@ -265,7 +265,7 @@ Ledger::Ledger (Ledger const& ledger,
 
 // Create a new open ledger that follows this one
 Ledger::Ledger (open_ledger_t, Ledger const& prevLedger,
-    NetClock::time_point closeTime)
+    TimeKeeper::time_point closeTime)
```

Comment:
Should be `NetClock::time_point`. Public interfaces need to use `NetClock` not `TimeKeeper`.